### PR TITLE
how security code reviews work

### PR DESCRIPTION
**Pull request by chayim**

### CODE REVIEW

**Review by ElizabethStirling — APPROVED** [review]
Seems like this will be a lot of work. I agree that it's the right work to do, but if we end up doing a lot of these, it will lead to other deadlines slipping somewhat. How frequently do we expect to be doing security code reviews?

File: handbook/engineering/security/index.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ After the each release, we hold a [retrospective](https://about.sourcegraph.com/
 
 ### Working with other teams
 
-The security team helps other teams investigate, orient, review, and sometimes plan changes.
+The security team helps other teams investigate, orient, review, and sometimes plan changes. Teams [request security code reviews](secure-code-review.md)
```
Comment (unknwon) [inline handbook/engineering/security/index.md:68]: nit
```suggestion
The security team helps other teams investigate, orient, review, and sometimes plan changes. Teams [request security code reviews](secure-code-review.md).
```

**Review by unknwon — APPROVED** [review]

File: handbook/engineering/security/secure-code-review.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Requesting a security review
+
+Security code reviews differ from [peer code reviews](https://about.sourcegraph.com/handbook/engineering/code_reviews), as they are usually on large blocks of code, and after the fact. These reviews bring a second set of eyes, with dedicated focus to the problem.
```
Comment (nicksnyder) [inline handbook/engineering/security/secure-code-review.md:3]: Should security review architecture before the fact?

File: handbook/engineering/security/secure-code-review.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+## Conducting a review
+
+To request a review from the security team:
+
+1. Create a GitHub issue in the [security repository](https://www.github.com/sourcegraph/security-issues), and add the *securityreview* label. 
+    * Provide a clear description, and links to the code, using Sourcegraph. 
+    * Provide as much detail as possible for the specifics of what has changed, and where we should focus. 
+    * Share usage scenarios, data flows, and any reference material
+
+1. Share the issue in the Security team slack channel, asking for a review. If you have a deadline, share it. By default, we target reviews for the upcoming iteration.
```
Comment (nicksnyder) [inline handbook/engineering/security/secure-code-review.md:14]: Under what circumstances would a team have a deadline? How much lead time do you want?

Give teams some guidelines so they know how to think ahead.

File: handbook/engineering/security/secure-code-review.md (hunk 2 above)
Comment (chayim) [reply handbook/engineering/security/secure-code-review.md:3]: Agreed - and updated. I merged accidentally, but this (with clarification) is in main.

**Pull request merged**